pkg/didcomm/protocol/introduce/metadata.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package introduce
+
+import (
+	"github.com/google/uuid"
+
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/common/service"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/didexchange"
+)
+
+const (
+	// protocol state machine identifier
+	metaPsmID        = Introduce + "_psm_id"
```

Comment:
a play on this. PIID -> protocol instance ID

Reply:
Done!